src/gsy_e/models/strategy/external_strategies/smart_meter.py

```diff
@@ -374,8 +381,14 @@ def _offer_aggregator(self, arguments: Dict) -> Dict:
                     raise OrderCanNotBePosted("The smart meter did not produce enough energy, ",
                                               "settlement offer can not be posted.")
                 available_energy = self.state.get_unsettled_deviation_kWh(market.time_slot)
-            else:
+            elif self.area.is_market_future(market.id):
+                available_energy = self.state.get_available_energy_kWh(
+                    str_to_pendulum_datetime(arguments["time_slot"]))
+            elif self.area.is_market_spot(market.id):
```

Comment:
Nit: the same comment as https://github.com/gridsingularity/gsy-e/pull/1537/files#r988908565 can be applied here.